e2e-tests/tests/e2e_flows.py

```diff
@@ -0,0 +1,310 @@
+import pytest
+import json
+
+from common.assertions import equal_json_strings
+from common.methods import analyze, anonymize, analyzer_supported_entities
+
+
+@pytest.mark.api
```

Comment:
the tests integrating with multiple services are planned to be marked with @pytest.mark.integration
We will run them on the build, and agains staging when deploying to prod